style/golf.mss

```diff
@@ -0,0 +1,62 @@
+#landcover[zoom >= 12] {
+  ::high-zoom[zoom >= 12] {
```

Comment:
Could you clarify what this means? Would you prefer all of these rules be moved to the `landcover.mss` file?

Reply:
No, just that if you use an attachment (the `::high-zoom`) that will be rendered outside the normal drawing order of the layer - which we don't want to happen here.

Reply:
I see. I need some help, then. Removing the attachment here makes these changes disappear. I specified `::high-zoom` because that was specified on the [existing `#landcover` layer](https://github.com/gravitystorm/openstreetmap-carto/blob/master/style/landcover.mss#L76-L77), and unless I directly place my new layers in `landcover.mss`, I'm not sure how to make sure these new styles are painted on top of the other landcover styles.

I see that in your [ac style](https://github.com/imagico/osm-carto-alternative-colors/blob/master/golf.mss) you do not have to specify an attachment, so maybe I just need help understanding how mapnik is ordering these layers internally.

Reply:
Oh, sorry, forgot about the fading, yes, you have to use the same attachment as the other landcover features then (and we should really get rid of the fading of course).

Reply:
Ah, thanks for the clarification.